src/org/sosy_lab/common/io/TempFile.java

```diff
@@ -9,6 +9,7 @@
 package org.sosy_lab.common.io;
 
 import static com.google.common.base.Preconditions.checkNotNull;
+import static com.google.common.io.MoreFiles.deleteRecursively;
```

Comment:
Static imports are useful for small and common methods like `checkNotNull`, but I would suggest to use them only when it actually makes the code better. For the single call to `MoreFiles.deleteRecursively()` it seems that without the import the code would be just as readable?